hazelcast-hibernate5/src/main/java/com/hazelcast/hibernate/CacheEnvironment.java

```diff
@@ -136,25 +156,17 @@ public static int getDefaultCacheTimeoutInMillis() {
         return (int) DEFAULT_CACHE_TIMEOUT.toMillis();
     }
 
-    public static int getCacheCleanupInSeconds(final Properties props) {
-        int delay = DEFAULT_CACHE_CLEANUP_DELAY;
+    public static Duration getCacheCleanupInSeconds(final Properties props) {
```

Comment:
Is `InSeconds` leftover here? It's been renamed in other modules and might be confusing if has no specific reason.

Reply:
Totally a leftover, fixing now